BUILD

```diff
@@ -0,0 +1,522 @@
+        "c++/src/capnp/serialize-packed.c++",
+        "c++/src/capnp/stream.capnp.c++",
+        "c++/src/capnp/stringify.c++",
+    ],
+    hdrs = [
+        "c++/src/capnp/any.h",
+        "c++/src/capnp/arena.h",
+        "c++/src/capnp/blob.h",
+        "c++/src/capnp/c++.capnp.h",
+        "c++/src/capnp/capability.h",
+        "c++/src/capnp/common.h",
+        "c++/src/capnp/compat/std-iterator.h",
+        "c++/src/capnp/dynamic.h",
+        "c++/src/capnp/endian.h",
+        "c++/src/capnp/generated-header-support.h",
+        "c++/src/capnp/layout.h",
+        "c++/src/capnp/list.h",
+        "c++/src/capnp/membrane.h",
+        "c++/src/capnp/message.h",
+        "c++/src/capnp/orphan.h",
+        "c++/src/capnp/pointer-helpers.h",
+        "c++/src/capnp/pretty-print.h",
+        "c++/src/capnp/raw-schema.h",
+        "c++/src/capnp/schema.capnp.h",
+        "c++/src/capnp/schema.h",
+        "c++/src/capnp/schema-lite.h",
+        "c++/src/capnp/schema-loader.h",
+        "c++/src/capnp/schema-parser.h",
+        "c++/src/capnp/serialize.h",
+        "c++/src/capnp/serialize-async.h",
+        "c++/src/capnp/serialize-packed.h",
+        "c++/src/capnp/serialize-text.h",
+        "c++/src/capnp/stream.capnp.h",
+    ],
+    copts = [
+        "-Wno-shadow",
+        "-Wno-unused-parameter",
+        "-iquote c++/src/kj",
+    ],
+    include_prefix = "capnp",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//:kj",
+        "//:kj-async",
+    ],
+)
+
+cc_import(
+    name = "capnp-rpc_schemas",
+    hdrs = [
+        "c++/src/capnp/persistent.capnp",
+        "c++/src/capnp/rpc.capnp",
+        "c++/src/capnp/rpc-twoparty.capnp",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "capnp-rpc",
+    srcs = [
+        "c++/src/capnp/capability.c++",
+        "c++/src/capnp/dynamic-capability.c++",
+        "c++/src/capnp/ez-rpc.c++",
+        "c++/src/capnp/membrane.c++",
+        "c++/src/capnp/persistent.capnp.c++",
+        "c++/src/capnp/rpc.c++",
+        "c++/src/capnp/rpc.capnp.c++",
+        "c++/src/capnp/rpc-twoparty.c++",
+        "c++/src/capnp/rpc-twoparty.capnp.c++",
+        "c++/src/capnp/serialize-async.c++",
+    ],
+    hdrs = [
+        "c++/src/capnp/ez-rpc.h",
+        "c++/src/capnp/persistent.capnp.h",
+        "c++/src/capnp/rpc.capnp.h",
+        "c++/src/capnp/rpc.h",
+        "c++/src/capnp/rpc-prelude.h",
+        "c++/src/capnp/rpc-twoparty.capnp.h",
+        "c++/src/capnp/rpc-twoparty.h",
+    ],
+    copts = [
+        "-Wno-shadow",
+        "-Wno-unused-parameter",
+    ],
+    include_prefix = "capnp",
+    visibility = ["//visibility:public"],
+    deps = [
+        ":capnp",
+        "//:kj",
+        "//:kj-async",
+    ],
+)
+
+cc_import(
+    name = "capnp-json_schemas",
+    hdrs = [
+        "c++/src/capnp/compat/json.h",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "capnp-json",
```

Comment:
There's both hyphens and underscores. Would we be able to make these all underscores for consistency?